test/unit/ajax.js

```diff
@@ -2327,6 +2327,15 @@ if ( typeof window.ArrayBuffer === "undefined" || typeof new XMLHttpRequest().re
 		} );
 	} );
 
+	// Selector should be trimmed to avoid leading spaces (#14773)
+	// Selector should include any valid non-HTML whitespace (#3003)
+	QUnit.asyncTest( "jQuery.fn.load( URL_SELECTOR with non-HTML whitespace(#3003) )", 1, function( assert ) {
+		jQuery( "#first" ).load( "data/test3.html   #whitespace\xA0 ", function() {
+			assert.strictEqual( jQuery( this ).children( "div" ).length, 1, "Verify that specific elements were injected" );
+			QUnit.start();
```

Comment:
Apologies for the broken links. Rule `no-async-test` should definitely catch this, let me know if I can help with integration.